docs/Architecture.md

```diff
@@ -93,7 +93,21 @@ Further details will be added to this section once https://github.com/strimzi/pr
 
 ### Transport layer
 
-Further details will be added to this section once https://github.com/strimzi/proposals/pull/6 and https://github.com/strimzi/proposals/pull/9 have been finalized.
+The Strimzi-ui makes use of both HTTP and WebSockets. This section details what each protocol is responsible for.
+
+#### HTTP
+
+Currently, HTTP is only used to retrieve static assets from the UI server.
+
+#### WebSockets
+
+[GraphQL](https://graphql.org/)
+
+All messages with GraphQL are done over a WebSocket, even standard queries. WebSockets are required by GraphQL to do subscriptions. Subscriptions allow the client to listen for data changes from the server so that it can stay updated in real-time. All other quieries are bundled into the same websocket so a clean network traffic log can be kept. The WebSocket will be held with the UI server, which will then proxy any websocket messages to the GraphQL admin server. This proxy allows for custom Apollo errors to be injected into the websocket so that the client can handle additional errors such as session expiration. The proxy is also responsible for injecting an authorization header for the client so that a connection can be etablished to the admin server. For full details of the WebSocket flow, [view our proposal](https://github.com/strimzi/proposals/blob/master/011-strimzi-ui.md#session-management-and-authentication).
```

Comment:
No objection to using websockets - will need to confirm if the server needs any change in the api module to proxy WS traffic

Reply:
Ok cool, yeah i'll use #55 to verify what needs to happen to the server to make this work

Reply:
My branch for #42 (https://github.com/ajborley/strimzi-ui/tree/42-client-side-logging) builds on Matt's experimental support for websocket handling:
https://github.com/ajborley/strimzi-ui/blob/42-client-side-logging/server/main.ts#L69-L79
https://github.com/ajborley/strimzi-ui/blob/42-client-side-logging/server/log/router.ts#L25-L34